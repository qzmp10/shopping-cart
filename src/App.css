.nav {
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding-right: 1rem;
  padding-left: 1rem;
}

.navigateContainer {
  display: flex;
  align-items: center;
  justify-content: space-between;
  width: 60%;
}

.home {
  margin-top: 40%;
  display: flex;
  flex-direction: column;
  align-items: center;
}

.header {
  font-size: 2.5rem;
}

.underHeader {
  font-size: 1.5rem;
}

.shopNow {
  margin-top: 1rem;
  font-size: 1.5rem;
  border: solid 1px black;
  padding: .5rem;
}