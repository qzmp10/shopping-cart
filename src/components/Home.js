
import React, { useState, useEffect, useRef } from 'react'
import { BrowserRouter, Routes, Route, Link } from 'react-router-dom'

export default function Home(props) {
    return (
        <>
            <div className='home'>
                <h1 className='header'>Homepage</h1>
                <div className='underHeader' role='second-heading'>Bla, bla bla.</div>
                <Link to='/products'>
                    <div className='shopNow'>Shop</div>
                </Link>
            </div>
        </>

    )
}