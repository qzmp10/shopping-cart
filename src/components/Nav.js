import React from 'react';

export default function Nav(props) {
    return (
        <div className='nav'>
            <h1 className='logo'>Logo</h1>
            <div className='navigateContainer'>
                <div className='navProducts'>Products</div>
                <div className='cart'>Cart</div>
                <div className='contactUs'>Contact Us</div>
            </div>
        </div>
    )
}