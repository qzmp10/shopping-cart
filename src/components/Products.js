import React, { useState, useEffect, useRef } from 'react'
import { BrowserRouter, Routes, Route, Link } from 'react-router-dom'

export default function Products() {
    return (
        <div> Helo world</div>
    );
}