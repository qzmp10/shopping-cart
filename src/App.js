import './App.css';
import React, { useState, useEffect, useRef } from 'react'
import Nav from './components/Nav';
import { BrowserRouter, Routes, Route, Link } from 'react-router-dom'
import Products from './components/Products';
import Home from './components/Home';


function App() {
  return (
    <BrowserRouter>
      <div className="App">
        <Nav />
        <Routes>
          <Route path='/' element={<Home />}></Route>
          <Route path='/products' element={<Products />}></Route>
        </Routes>
      </div>
    </BrowserRouter>
  );
}

export default App;
