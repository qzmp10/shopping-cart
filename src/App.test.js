import { render, screen } from '@testing-library/react';
import App from './App';
import Nav from './components/Nav';
import Products from './components/Products';
import Home from './components/Home'

describe("nav bar", () => {
  it('nav bar matches snapshot', () => {
    const {navBar} = render (<Nav/>)
    expect(navBar).toMatchSnapshot()
  }) // no clue what this does to be honest


  it()
}) 
