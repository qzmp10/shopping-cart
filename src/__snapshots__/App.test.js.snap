// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav bar nav bar matches snapshot 1`] = `undefined`;
